snuba/web/rpc/v1/resolvers/R_eap_items/resolver_trace_item_table.py

```diff
@@ -184,7 +184,32 @@ def _get_reliability_context_columns(
     """
     extrapolated aggregates need to request extra columns to calculate the reliability of the result.
     this function returns the list of columns that need to be requested.
+
+    If alias_prefix is provided, it will be prepended to the alias of the returned columns.
     """
+
+    if column.HasField("formula"):
+        # also query for the left and right parts of the formula separately
+        # this will be used later to calculate the reliability of the formula
+        # ex: SELECT agg1/agg2 will become SELECT agg1/agg2, agg1, agg2
+        leftcolumn = Column()
+        leftcolumn.CopyFrom(column.formula.left)
+        rightcolumn = Column()
+        rightcolumn.CopyFrom(column.formula.right)
+        leftselect = SelectedExpression(
+            name=leftcolumn.label,
+            expression=_column_to_expression(leftcolumn, request_meta),
+        )
+        rightselect = SelectedExpression(
+            name=rightcolumn.label,
+            expression=_column_to_expression(rightcolumn, request_meta),
+        )
```

Comment:
Given that a `Column` can also be a formula, wouldn't you only want to add these columns if the column is an aggregation? otherwise you risk doing something like:

`a / (b * c)` -> `a`, `(b * c)`, `b`, `c` 

In this case `b * c` is redundant